bin/webwork3

```diff
@@ -10,5 +10,12 @@ use Mojo::File qw(curfile);
 use lib curfile->dirname->sibling('lib')->to_string;
 use Mojolicious::Commands;
 
+# check if the config file has been created
+my $webwork_dir = curfile->dirname->dirname("..");
+
+warn qq!The file $webwork_dir/conf/webwork3.yml does not exist.
+	Perhaps you haven't copied webwork3.dist.yml to this file.!
```

Comment:
```suggestion
	Perhaps you haven't copied webwork3.yml.dist to this file.!
```
The dist file is `webwork3.yml.dist`.  Should we change the file name to `webwork3.dist.yml`?  The advantage of putting the `yml` extension at the end is that it will be recognized as a YAML file by text editors and linters and such.